examples/run.py

```diff
@@ -1,21 +1,64 @@
-import sys
+import argparse
 
 from cleanvision.imagelab import Imagelab
 
 if __name__ == "__main__":
-    dataset_path = sys.argv[1]
+    parser = argparse.ArgumentParser(description="Demonstrates how to use Imagelab")
+    parser.add_argument("--path", type=str, help="path to dataset", required=True)
+    args = parser.parse_args()
 
-    # Run Imagelab with default settings
-    imagelab = Imagelab(dataset_path)
-    imagelab.list_default_issue_types()
-    imagelab.find_issues()
+    dataset_path = args.path
+
+    """
+    Example 1
+
+    This example demonstrates using Imagelab to
+    1. Explore the dataset
+    2. Find all types of issues
+    3. Report top issues found and visualize them
+    4. Check Imagelab attributes: imagelab.issue_summary, imagelab.issues, imagelab.info
```

Comment:
maybe could just summarize it as:

```This example demonstrates the default Imagelab workflow to detect various types of issues in an image dataset.```

I think the details are not particularly helpful to list here since they are vague without the code (especially the purpose of the attributes)